### ISSUE TITLE
Gallery access button.

### ISSUE THREAD

**Issue by moshpirit** [hannesmannerheim/qvitter#201]
It would be incredibly good to have some kind of "Gallery" that shows just the pictures and videos shared (and maybe included external videos and pictures). Something like the [Quitim interface](https://github.com/hannesmannerheim/quitim) (I don't know if it shows external thumbnails too).

**Comment by hannesmannerheim** [comment]
agree. but for it to be useful i think we need to make gnusocial download remote attachments and create thumbnails for them. i'll see if i can accomplish that within the qvitter plugin.

**Comment by mmn** [comment]
@hannesmannerheim  If it's just a function that will download remote files and thumbnail them, it could probably be done in a separate plugin since it would benefit both classic and Qvitter layouts. :)

I'm onto that part though.

**Comment by hannesmannerheim** [comment]
that would be great :)

are you planning on deleting the full-size image and just keeping the thumbnail? becaulse qvitter needs the full size image to exist so it can create new larger tumbnails

**Comment by mmn** [comment]
I think it should keep the full sized image (expropriate and don't care about copyright!) with a configurable maximum-downloadable-file-size.

**Comment by hannesmannerheim** [comment]
awesome :) 
it's not really a copy, it's a _cache_ ;)
